tests/_data/project_mock/src/Pyz/Zed/TestIntegratorWirePlugin/TestIntegratorWirePluginDependencyProvider.php

```diff
@@ -138,4 +138,21 @@ public function getTestArrayMergePlugins(): array
     {
         return array_merge([\ArrayObject::class]);
     }
+
+    /**
+     * @return array
+     */
+    protected function getWrappedPlugins(): array
```

Comment:
pls add case if getWrappedPlugins method doesn't exist in child.

in `TestParentIntegratorWirePluginDependencyProvider`
protected function getWrappedPlugins()
{return [];} 
and add plugins stack
but probably this case for previous story

Reply:
@DmytroKlymanSpryker probably we need to move it to another ticket. the case when methods exist only in parent method. for `call` seems like it's not actual case

Reply:
I'll create another ticket for that, but we have higher priorities at the moment